extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -173,9 +179,6 @@ export class AmpImageViewer extends AMP.BaseElement {
 
   /** @override */
   resumeCallback() {
-    if (this.sourceAmpImage_) {
```

Comment:
This is unnecessary. If the image-viewer is laid out, we will have already captured its dimensions and do not need to do so again. If the image-viewer has not been laid out, the resumeCallback doesn't need to do anything.